src/vg/civcraft/mc/citadel/command/commands/Fortification.java

```diff
@@ -62,12 +62,12 @@ public boolean execute(CommandSender sender, String[] args) {
 		}
 		
 		PlayerType type = g.getPlayerType(uuid);
-		if (type == null){
+		if (!p.hasPermission("citadel.admin") && !p.isOp() && type == null){
```

Comment:
can you change this to !(p.hasPermission("citadel.admin") || p.isOp()) && type == null